README.md

```diff
@@ -14,6 +14,15 @@ tweetnacl.h -> 02aee579efbe4f12ad8ffb23c6772841
 ## Fixes
 Change integer size in commit [b98e0ca](https://github.com/RIOT-OS/tweetnacl/commit/b98e0ca8beb9b802b5c0492d88846bd9de9946eb)
 
+Fix a warning on line 58 about comparison between two different integer
+sizes.
```

Comment:
The `u32` variable `d` remained the same, the change to `int` it's because the argument on the function is int. I didn't designed the prototype of this function, but in my opinion it should match the argument to prevent overflow.